ci-operator/step-registry/osd/create/create/osd-create-create-commands.sh

```diff
@@ -115,10 +116,15 @@ echo '{ "expiration_timestamp": "'"${INIT_EXPIRATION_DATE}"'" }' | ocm patch "/a
 echo -n "${CLUSTER_ID}" > "${HOME}/cluster-id"
 
 echo "Waiting for cluster ready..."
+start_time=$(date +"%s")
 while true; do
   sleep 60
   CLUSTER_STATE=$(ocm get cluster "${CLUSTER_ID}" | jq -r '.status.state')
   echo "Cluster state: ${CLUSTER_STATE}"
+  if (( $(date +"%s") - $start_time >= $CLUSTER_TIMEOUT )); then
+    echo "error: Timed out while waiting for cluster to be ready"
+    exit 1
+  fi
```

Comment:
I would move this if block after the check `[[ "${CLUSTER_STATE}" == "ready" ]]`
Just in case we hit limit CLUSTER_TIMEOUT (1.5 hour ATM) but cluster is finally ready :-)

Reply:
Good point, thanks! I'll change it now